src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -529,6 +544,15 @@
   (with-open [r (reader pem)]
     (SSLUtils/pemToCRLs r)))
 
+(schema/defn ^:always-validate pem->ca-crl :- X509CRL
+  "Given a CRL chain and a CA Cert, extract the first CRL"
+  [crl-chain :- Readerable]
```

Comment:
I don't see a CA cert being given here, but maybe I'm missing something?

Reply:
Whoops, my initial implementation mirrored the `pem->ca-cert` with the same error handling, but after implementing I realized that the CA cert is not readily available when using the crl. I'll update this docstring and the commit message.